app/main.py

```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    count = 0
+    for i in phrase.lower():
+        if i.lower() == letter.lower():
+            count += 1
+    return count
```

Comment:
1. The solution is ok, but can you try to rewrite it using just 1 line?
2. Always add a blank line at the end of the file.